pom.xml

```diff
@@ -66,7 +66,8 @@
     <dependency.commons-configuration.commons-configuration.version>1.9</dependency.commons-configuration.commons-configuration.version>
     <dependency.commons-collections.commons-collections.version>3.2.1</dependency.commons-collections.commons-collections.version>
     <dependency.org.slf4j.slf4j-api.version>1.7.7</dependency.org.slf4j.slf4j-api.version>
-    <dependency.org.mockito.mockito-all.version>1.9.5</dependency.org.mockito.mockito-all.version>
+    <dependency.org.mockito.mockito-all.version>1.9.5-rc1</dependency.org.mockito.mockito-all.version>
```

Comment:
Just curious as to why this has been changed to an rc.

Reply:
And as to why this version specifically, no reason other than this combination of mockito and powermock versions caused no other unrelated test failures, while other combos (I tried several, including the latest for both) caused many test failures. Knowing the history of those two projects and that not all versions are compatible, I didn't think it was worth spending time on root causing those unrelated failures, especially since all tests are happy with this combination.